helm_deploy/hmpps-prisoner-to-nomis-update/values.yaml

```diff
@@ -80,6 +80,7 @@ generic-service:
     FEATURE_EVENT_INCENTIVES_PRISON_LEVEL_CHANGED: true
     FEATURE_EVENT_DELETEALL: false
     FEATURE_EVENT_COURT_SENTENCING_COURT_CASE_CREATED: true
+    FEATURE_EVENT_COURT_SENTENCING_COURT_APPEARANCE_CREATED: true
```

Comment:
events not defined in cloud platform yet, the names will almost certainly change